daemon/k8s_watcher.go

```diff
@@ -1256,25 +1256,23 @@ func (d *Daemon) addK8sSVCs(svcID k8s.ServiceID, svc *k8s.Service, endpoints *k8
 
 		uniqPorts[fePort.Port] = false
 
-		if fePort.ID == 0 {
-			feAddr := loadbalancer.NewL3n4Addr(fePort.Protocol, svc.FrontendIP, fePort.Port)
-			feAddrID, err := service.AcquireID(*feAddr, 0)
-			if err != nil {
-				scopedLog.WithError(err).WithFields(logrus.Fields{
-					logfields.ServiceID: fePortName,
-					logfields.IPAddr:    svc.FrontendIP,
-					logfields.Port:      fePort.Port,
-					logfields.Protocol:  fePort.Protocol,
-				}).Error("Error while getting a new service ID. Ignoring service...")
-				continue
-			}
-			scopedLog.WithFields(logrus.Fields{
-				logfields.ServiceName: fePortName,
-				logfields.ServiceID:   feAddrID.ID,
-				logfields.Object:      logfields.Repr(svc),
-			}).Debug("Got feAddr ID for service")
-			fePort.ID = loadbalancer.ServiceID(feAddrID.ID)
+		feAddr := loadbalancer.NewL3n4Addr(fePort.Protocol, svc.FrontendIP, fePort.Port)
```

Comment:
what about when a service is added through the cilium-agent's CLI?

Reply:
You can specify it via the `--id` param. E.g. `cilium service update ... --id 1234`.